docs/docs-content/clusters/data-center/nutanix/install-pcg.md

```diff
@@ -0,0 +1,200 @@
+---
+sidebar_label: "Install Private Cloud Gateway"
+title: "Install Private Cloud Gateway"
+description: "Learn how to install a Nutanix Private Cloud Gateway in Palette."
+hide_table_of_contents: false
+sidebar_position: 10
+toc_min_heading_level: 2
+toc_max_heading_level: 3
+tags: ["data center", "nutanix"]
+---
+
+
+A Private Cloud Gateway (PCG) is required to connect your Nutanix cloud with Palette. The PCG enables Palette to securely monitor Nutanix clusters in the deployed cloud environment. This section guides you on how to install the PCG.
+
+## Prerequisites
+
+- A Nutanix Prism Central account with *Prism Admin* role. 
+
+- A Nutanix subnet created in Nutanix Prism Central.
+
+- A Nutanix cloud registered with Palette. For information about registering your cloud, review [Register Nutanix Cloud](register-nutanix-cloud.md).
+
+- A Nutanix Cluster API (CAPI) OS image. For guidance on creating the image, refer to [Building CAPI Images for Nutanix Cloud Platform](https://image-builder.sigs.k8s.io/capi/providers/nutanix.html#building-capi-images-for-nutanix-cloud-platform-ncp).
+
+- The following applications installed. The [Common Prerequisites](https://cluster-api.sigs.k8s.io/user/quick-start#common-prerequisites) section of the [Install clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start#install-clusterctl) page provides links to the applications below.
+
+  - [Docker](https://docs.docker.com/engine/install/)
+  - [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation) or  [kind](https://kind.sigs.k8s.io/) ??
+  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
+  - [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start#install-clusterctl) 
+
+
+## Setup
```

Comment:
OK, so there is a misunderstanding about PCG. Nutanix (and custom clouds) requires a self-hosted PCG. What is described below looks mandatory while it's not: it's only one of the option, we offer the option to have any type of existing kubernetes cluster to run the PCG.

The only requirement is that it must a network connectivity w/ Prism.

Reply:
Revised as:

"- A Kubernetes cluster created that has network connectivity with Nutanix Prism. This cluster will connect the Nutanix cloud with Palette via the PCG. Various types of Kubernetes clusters can be used for the PCG. However, the [Setup](#setup) section in this guide uses the process described in the [Nutanix Getting Started](https://opendocs.nutanix.com/capx/v1.1.x/getting_started/) resource and its specified [Common Prerequisites](https://cluster-api.sigs.k8s.io/user/quick-start#common-prerequisites). This method requires installing the following applications:

  - [Docker](https://docs.docker.com/engine/install/)
  - [kind](https://kind.sigs.k8s.io/docs/user/quick-start/#installation)
  - [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl-linux/)
  - [clusterctl](https://cluster-api.sigs.k8s.io/user/quick-start#install-clusterctl) ""